breathecode/registry/views.py

```diff
@@ -614,6 +614,8 @@ def get(self, request, asset_slug=None):
 
         if 'big' in self.request.GET:
             serializer = AssetMidSerializer(items, many=True)
+        elif 'with_techs' in self.request.GET:
```

Comment:
should be better use the OData standard, `?expand=RELATED_FIELD_NAME1,RELATED_FIELD_NAME2`

https://www.odata.org/getting-started/basic-tutorial/#expand